README.md

```diff
@@ -194,19 +194,7 @@ k apply -f audius/creator-node/creator-node-pvc.yaml
 k apply -f audius/creator-node/creator-node-deploy-ipfs.yaml
 ```
 
-3.) Before deploying Creator Node backend, we must obtain IPFS running host IP and service nodePort, so we can pass that config to Creator Node.
-
-> NOTE If you only have an "InternalIP", ensure your cluster node has an externally accessible network interface
-
-```
-# IPFS_CLUSTER_IP
-kubectl get node $(kubectl -n default get pod -l release=creator-node,tier=ipfs -o=jsonpath='{.items[0].spec.nodeName}') -o=jsonpath='{.status.addresses[?(@.type=="ExternalIP")].address}'
-
-# IPFS_CLUSTER_PORT
-kubectl -n default get svc creator-node-ipfs-svc -o=jsonpath='{.spec.ports[?(@.name=="swarm")].nodePort}'
-```
-
-4.) Update Creator Node backend config map with the env vars. The  full list of env vars and explanations can be found on the wiki [here](https://github.com/AudiusProject/audius-protocol/wiki/Content-Node-%E2%80%90-How-to-run#required-environment-variables)
```

Comment:
I think we should move Point 3 from under https://github.com/AudiusProject/audius-k8s-manifests#6-security--infrastructure-configuration to here, so that someone setting up for the first time doesn't miss the IPFS related setup.

Reply:
yeah thats not a bad idea. i've actually been trying to think of a way to automate the ipfs announce check so we can do it with a health check